src/qt/diagnosticsdialog.cpp

```diff
@@ -43,7 +43,7 @@ int DiagnosticsDialog::VerifyBoincPath() {
 }
 
 int DiagnosticsDialog::FindCPID() {
-    std::string cpid = GetArgument("cpid", "");
+    std::string cpid = GetArgument("PrimaryCPID", "");
 
     if(cpid.length() != 0)
```

Comment:
We should use `IsResearcher` here, right?

Reply:
good one. done